api/Crypto.json

```diff
@@ -142,16 +140,10 @@
             "ie": {
               "version_added": false
             },
-            "nodejs": [
-              {
-                "version_added": "19.0.0"
-              },
-              {
-                "version_added": "16.7.0",
-                "version_removed": "19.0.0",
-                "notes": "From version 14.17.0, a similar method is available as <a href='https://nodejs.org/docs/latest-v14.x/api/crypto.html#crypto_crypto'><code>crypto.randomUUID()</code></a>."
-              }
-            ],
+            "nodejs": {
```

Comment:
Why don't we keep the `"version_added": "19.0.0"` entry?

Reply:
because there is no difference made to `randomUUID()` before v19.0.0 and after v19.0.0 (while the `Crypto` interface is exposed to global scope, so just combine the previous two entries)